example/hands-on/create-hands-on-app.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+########################
+# rn-spoilerをテンプレートとして、学習コンテンツの内容を適用したアプリの作成
+# 第一引数：アプリ名(Optional. デフォルト値:HandsOnApp)
+########################
+
+set -euo pipefail
+APP_NAME=${1:-HandsOnApp}
+
+echo "Start creating $APP_NAME."
+
+# 実行ファイルのディレクトリを取得
+EXECUTABLE_SCRIPT_DIR=$(cd $(dirname $0); pwd)
+# プロジェクトルートを取得
+ROOT_DIR=$(cd $EXECUTABLE_SCRIPT_DIR; git rev-parse --show-toplevel)
+# パッチファイルが格納されているディレクトリを取得
+PATCHES_DIR=$EXECUTABLE_SCRIPT_DIR/patches
+# `generated`ディレクトリの中に新規アプリを作成する 
+WORK_DIR=$ROOT_DIR/generated
+mkdir -p $WORK_DIR
+# 新規アプリのルートディレクトリ
+APP_ROOT_DIR=$WORK_DIR/$APP_NAME
+
+cd $WORK_DIR
+# すでにアプリが存在していた場合は削除
+rm -rf $APP_NAME
+
+# rn-spoilerをテンプレートとして、新規アプリを作成
+npx react-native init --template https://github.com/ws-4020/rn-spoiler.git $APP_NAME
```

Comment:
現状、npm固定にしておきたい。（`package-lock.json`前提にもなってるし）

```suggestion
npx react-native init --npm --template https://github.com/ws-4020/rn-spoiler.git $APP_NAME
```